snl_shell/snl.c

```diff
@@ -147,6 +151,18 @@ int snl_detect_pipe(char** args){
   return 0; // no pipe
 }
 
+int snl_detect_redirect(char** args) {
+  for (int i = 0; i < MAXARGS; i++) { //TODO: make this not maxargs
+    if(args[i] == NULL){
+      return 0; // reached end, no pipe
+    }
+    if (strcmp(args[i],">") == 0){
+      return i; // yes pipe
+    }
+  }
+  return 0; // no pipe
+}
+
```

Comment:
i would add some error checking to make sure the user doesn't input both | and </>